File: app/src/main/java/foo/org/gitapp/models/Repository.java
package foo.org.gitapp.models;

/**
 * Created by enoks on 22.2.2017.
 */

public class Repository {

    public class Owner {

        private String login;

        public String getLogin() {
            return login;
        }

        public void setLogin(String login) {
            this.login = login;
        }

    }

    private String id;
    private String name;
    private Owner owner;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public Owner getOwner() {
        return owner;
    }

    public void seOwner(Owner owner) {
        this.owner = owner;
    }

}

File: app/src/main/java/foo/org/gitapp/models/Repository.kt
package foo.org.gitapp.models

/**
 * Created by enoks on 22.2.2017.
 */

class Repository {

    inner class Owner {

        var login: String? = null

    }

    var id: String? = null
    var name: String? = null
    var owner: Owner? = null
        private set

    fun seOwner(owner: Owner) {
        this.owner = owner
    }

}
